WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -185,8 +188,15 @@ nasrani = {
 	
 	#caste_tier = 3 #Conflicts with brahmin
 
+	tolerates_christian = yes
+	tolerates_indian_group = yes
+
```

Comment:
That was intentional. Nasranis live in India and are constantly exposed to Indian cultures, and vice-versa. Indian religions accept them as an unique caste and even consider them prestigious, as the tooltip and descriptions say.